packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -138,10 +116,13 @@ export default function FeedItemComponent({
           onUpvoteClick={(post, upvoted) =>
             onUpvote(post, index, row, column, upvoted)
           }
-          onLinkClick={(post) => onPostClick(post, index, row, column)}
+          onLinkClick={(post, e) => onPostClick(post, index, row, column, e)}
           onBookmarkClick={(post, bookmarked) =>
             onBookmark(post, index, row, column, bookmarked)
           }
+          onReadArticleClick={(e) =>
+            onPostClick(item.post, index, row, column, e, true)
```

Comment:
I'm not sure we want to track it as a view at the moment. But let me get back to you on that

Reply:
We want to flag an article as read-only when the user actually visits the article

Reply:
Oh, cool. Then this should count as read since they will be redirected to the actual article, right?

With that, I will label the event based on our taxonomy as 'go to link' but register the `view` that the user has made. WDYT?

Reply:
By "register as view", what do you mean exactly? Because views are automatically counted once you visit the redirector link. Do you refer to the local weekly badge count?

Reply:
Yes, I mean we add an entry in our database at `view` table when the post link is opened (clicked the card), and with the `ReadArticle` button, it opens the link (but not through the whole card click anymore), so I am thinking it should perform the same things and marks the post as already read by the user.

I think the read article button inside post modal button doesn't create a new entry in our database whenever it is clicked though it contains all the parameters for tracking the post. That's why I am wondering if we should create an entry on the new card read article button, and the post modal's read article button should behave the same way?

Reply:
Or wait. I think I am actually mistaken. It seems we register view in the redirector, and we just locally update the cache about clicking a post card link.

Reply:
You're right. The redirector counts automatically the views and it's just the local cache that you update on article view